threedi_modelchecker/migrations/env.py

```diff
@@ -0,0 +1,28 @@
+from alembic import context
+from threedi_modelchecker.threedi_model.models import Base
+
+target_metadata = Base.metadata
+config = context.config
+
+
+def run_migrations_online():
+    """Run migrations in 'online' mode.
+
+    Note: SQLite does not (completely) support transactions, so, backup the
+    SQLite before running migrations.
+    """
+    connectable = config.attributes.get('connection')
+
+    with connectable.connect() as connection:
+        context.configure(
+            connection=connection, target_metadata=target_metadata
+        )
+
+        with context.begin_transaction():
+            context.run_migrations()
+
+
+if context.is_offline_mode():
```

Comment:
What is the 'offline' mode? I guess this has nothing to do with an internet connection.... ;)

Reply:
The offline mode generates SQL als plain text, so that you can run it later on a database. I don't see a use case for us however.